keyboards/boardsource/unicorne/config.h

```diff
@@ -0,0 +1,26 @@
+// Copyright 2023 jack (@waffle87)
+// SPDX-License-Identifier: GPL-2.0-or-later
+#pragma once
+
+#define RGB_MATRIX_LED_COUNT 54
+#define RGB_MATRIX_DEFAULT_MODE RGB_MATRIX_ALPHAS_MODS
+#define RGB_DISABLE_WHEN_USB_SUSPENDED
+
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+
+#define SPLIT_LAYER_STATE_ENABLE
+
+#define I2C_DRIVER I2CD1
+#define I2C1_SDA_PIN GP22
+#define I2C1_SCL_PIN GP23
+
+#define AUDIO_PIN GP28
+#define AUDIO_PWM_DRIVER PWMD6
+#define AUDIO_INIT_DELAY 1000
```

Comment:
Setting is a boolean, not a time setting.
```suggestion
#define AUDIO_INIT_DELAY
```